src/Zicht/Bundle/AdminBundle/Command/ExportSqlCommand.php

```diff
@@ -28,6 +29,7 @@ class ExportSqlCommand extends ContainerAwareCommand
         'xmlexcel'  => 'Exporter\\Writer\\XmlExcelWriter',
         'json'      => 'Exporter\\Writer\\JsonWriter',
         'csv'       => 'Exporter\\Writer\\CsvWriter',
+        'twig'      => 'Zicht\Bundle\AdminBundle\Exporter\Writer\TwigWriter',
```

Comment:
Officially double backslashes `\\` and this is also consistent with previous lines